client/src/components/layouts/DashboardSidebarData.js

```diff
@@ -18,13 +17,18 @@ import { useSelector } from 'react-redux';
 /*
     List of navigation inside the sidebar. Change the link to Nav.Link either here or in Sidebar.js.
 */
-const DashboardSidebarData = ({ setLogoutAlert }) => {
+const DashboardSidebarData = () => {
   const sidebarLinks = [
     {
       title: 'Dashboard',
       icon: <BsFillGridFill />,
       link: '/dashboard',
     },
+    {
+      title: 'Account',
+      icon: <BsPerson />,
+      link: '/dashboard/profile',
+    },
```

Comment:
I was thinking about removing the `Account` from the sidebar since we've already added the account button in the dropdown menu.
What do you guys think?

Reply:
Yes I agree, we should remove Account from sidebar

Reply:
Removed the Account from Sidebar as requested.